src/sentry/static/sentry/app/components/button/index.jsx

```diff
@@ -196,29 +197,29 @@ const getColors = ({priority, disabled, borderless, theme}) => {
   `;
 };
 
-const customProps = ['external', 'size'];
-const shouldForwardProp = p =>
-  (p !== 'disabled' && isPropValid(p)) || customProps.includes(p);
-
 const StyledButton = styled(
-  ({external, ...props}) => {
+  React.forwardRef((prop, ref) => {
```

Comment:
I know that `prop` is not the conventional name, but eslint was complaining about missing prop validation when the variable was called props and adding `StyledButton.propTypes` did not appease it.

Reply:
a comment saying something like that might be helpful, as it did throw me a bit